forc/src/cli/commands/add.rs

```diff
@@ -0,0 +1,37 @@
+use crate::ops::forc_edit;
+use anyhow::Result;
+use clap::Parser;
+// A utility for managing forc dependencies from the command line.
+
+/// Add or remove dependencies to a Forc.toml manifest file.
+#[derive(Debug, Parser)]
+#[clap(version)]
+#[clap(setting = clap::AppSettings::DeriveDisplayOrder)]
+#[clap(after_help = "\
+Example:
+  $ forc add ./crate/parser/
+  $ forc remove ./crate/parser/
+")]
+#[clap(override_usage = "\
+    forc add [OPTIONS] <DEP_PATH> ...
+    forc remove [OPTIONS] <DEP_PATH> ...
+")]
+pub struct Command {
+    /// Reference to a package to add or remove as a dependency
+    ///
+    /// You can reference a packages by:
+    /// - `<path>`, like `forc add ./crates/parser/` or `forc remove ./crates/parser/`
+    ///
+    #[clap(value_name = "DEP_ID")]
+    pub crates: Vec<String>,
```

Comment:
One small suggestion that I could make here would be maybe using an enum that could hold either an url or path. Because dependencies could be added as path or git (or registry index as we are starting to plan that) so in terms of code quality/maintainability it could make sense to have an enum of possible sources and we can then have different add/remove logic based on the enum type. I am not sure how well clap would interact with an enum but it might be worth playing with 👍